homeassistant/components/plugwise/quality_scale.yaml

```diff
@@ -0,0 +1,65 @@
+rules:
+  ## Bronze
+  config-flow: done
+  test-before-configure: done
+  unique-config-entry: done
+  config-flow-test-coverage: done
+  runtime-data: done
```

Comment:
`test_init.py` line 81: `    assert not hass.data.get(DOMAIN)`

We should not touch internals in tests and since this is moved to entry runtime data, this check is irrelevant